### DISCUSSION TITLE
New Discussion

### DISCUSSION THREAD

**Discussion by mikeng9944** [karleeov/chatgpt-aoai#5] [Q&A] [answered]
Why choose clerk as the login

**Comment by karleeov** [comment]
what is clear means?

**Reply by mikeng9944** [reply]
Why choose clerk as the login

**Reply by karleeov** [reply, marked as answer]
it can save you time to handle the login and register for users if you need more control and looking to manage to cooporate level you can consider to setup as azure ad
